src/encoded/ingestion/processors.py

```diff
@@ -150,11 +151,14 @@ def handle_metadata_bundle(submission: SubmissionFolio):
         project = get_parameter(submission.parameters, 'project')
         validate_only = get_parameter(submission.parameters, 'validate_only', as_type=bool, default=False)
 
+        submission_type = 'family_history' if submission.ingestion_type == 'family_history' else 'accessioning'
```

Comment:
See comment above, but unless there is a strong reason, I'd like to see this use just `'accession'` instead of `'accessioning'`.

This also needs a comment that explains what's going on. And I think if it's really just making synonyms of those two, it's better to write the conditional a different way so if you add other modes they are naturally covered. I think this is what you want (untested, and not yet incorporating my `'accession'` suggestion):
```suggestion
        # This rewrites the obsolete term `metadata_bundle` using one of two preferred modes,
        # 'accession' or 'family_history'.
        submission_type = 'accessioning' if submission.ingestion_type == 'metadata_bundle' else submission_type
```